src/shared/jira/types.ts

```
@@ -0,0 +1,276 @@
+  application: {
+    // "com.atlassian.jira",
+    type: string;
+    // "JIRA | TCS Bank"
+    name: string;
+  };
+  relationship: string;
+  object: {
+    url: string;
+    // "SMEDE-5372",
+    title: string;
+    icon: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)